docs/ops/image/Interpolate_4.md

```diff
@@ -219,7 +219,7 @@ class InterpolateCalculation:
         self.coordinate_transformation_mode = attrs.get('coordinate_transformation_mode', 'half_pixel')
         self.nearest_mode = attrs.get('nearest_mode', 'round_prefer_floor')
         self.cube_coeff = attrs.get('cube_coeff', -0.75)
-        self.antialias = attrs.get('antialias', False)
```

Comment:
Please, revert changes in the Python code. Since it is a Python the values should be `True` or `False` and not `true` or `false`

Reply:
Will change that, seems like I tried to be too clever